project/default.properties

```diff
@@ -1,2 +1,3 @@
-name=hello-akka-java
-description=This is a seed project and tutorial which creates a basic Java based Akka application.
+name = akka-quickstart-java
+description = Akka is a toolkit and runtime for building highly concurrent, distributed, and fault tolerant event-driven apps. This simple application will get you started building Actor based systems with Java.
+verbatim = *.xml
```

Comment:
Actually, the `build.gradle` file is not filtered out by `g8` so there is no need for any specific setting for it.